Framework/CurveFitting/src/Algorithms/FitPowderDiffPeaks.cpp

```diff
@@ -3009,8 +3008,9 @@ void FitPowderDiffPeaks::cropWorkspace(double tofmin, double tofmax) {
     g_log.error(errmsg.str());
     throw std::runtime_error(errmsg.str());
   } else {
-    cout << "[DBx211] Cropped Workspace Range: " << m_dataWS->x(m_wsIndex)[0]
-         << ", " << m_dataWS->x(m_wsIndex).back() << '\n';
+    g_log.error() << "[DBx211] Cropped Workspace Range: "
```

Comment:
I'm not sure `error` is the right level here? Each time this algorithm is run with the crop option the users would get something that looks like an error. It looks like an informational statement so I think `.debug` or `.information` would be more suited.